lib/system/excpt.nim

```diff
@@ -445,6 +445,12 @@ proc getStackTraceEntries*(e: ref Exception): seq[StackTraceEntry] =
   else:
     result = move(e.trace)
 
+proc getStackTraceEntries*(): seq[StackTraceEntry] =
```

Comment:
Can't you use the existing ``proc getStackTraceEntries*(e: ref Exception): seq[StackTraceEntry]``? I don't like yet-another system.nim extension.

Reply:
No, I can't. I don't have an exception.

Reply:
Hmm ok. Sucks.